src/scripts/main.js

```diff
@@ -1,3 +1,50 @@
 'use strict';
 
-// write code here
+const body = document.querySelector('tbody');
+const people = [];
+
+const getNumber = (string) => {
+  return parseFloat(string.replace('$', ''));
+};
+
+for (let i = 0; i < body.children.length; i++) {
+  const [ namePerson, position, age, salary ] = body.children[i].children;
+
+  people.push({
+    name: namePerson.innerText,
+    position: position.innerText,
+    age: age.innerText,
+    salary: salary.innerText,
+  });
+}
+
+const functionFoSort = (string) => {
+  if (string === 'Name' || string === 'Position') {
+    return (a, b) => a[string.toLowerCase()]
+      .localeCompare(b[[string.toLowerCase()]]);
+  }
+
+  if (string === 'Age' || string === 'Salary') {
+    return (a, b) =>
+      getNumber(a[string.toLowerCase()]) - getNumber(b[string.toLowerCase()]);
+  }
```

Comment:
I see you can return like this (example)
```suggestion
  return (a, b) => {
    const firstElement = a[string.toLowerCase()]
    const secondElement = b[string.toLowerCase()]
    
    if (string === 'Name') {
      return firstElement.localeCompare(secondElement)
    }
    
    if (string === 'Age) {
      return firstElement - secondElement
    }
  }
```

Reply:
Just to follow DRY principle (Don't Repeat Yourself)

Reply:
This conversation is commented at fe_chat by mentor